packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -3641,24 +3646,23 @@ export class ContainerRuntime
 		// We choose to close the summarizer after the snapshot cache is updated to avoid
 		// situations which the main client (which is likely to be re-elected as the leader again)
 		// loads the summarizer from cache.
-		if (this.summaryStateUpdateMethod === "restart") {
```

Comment:
Let's leave the if check but change it to something like `if (this.summaryStateUpdateMethod !== "refreshFromSnapshot") {` or whatever we want to call the old behavior that we'll remove.  Just to keep this super concise and not cut ourselves off from reverting prematurely (of course we don't imagine we'll need to, but why burn the bridge)

Reply:
Ok I see. yeah I'll do that.